core/modes/modes_not_dragon.talon

```diff
@@ -0,0 +1,41 @@
+mode: sleep
+not mode: sleep
```

Comment:
Is this saying "mode sleep or not mode sleep"? That is always true, right? Can't we just remove both of these lines?

Reply:
Without any modes specified, a talon file will be implicitly in command mode only. I don't think there's another way to accomplish this other than mode: all, which would become in full circle